docs/security/safeharbor.md

```diff
@@ -65,7 +65,11 @@ Aragon Voting was chosen because it provides a predictable, resilient, and timel
 
 Chain: eip155:1 (*Ethereum Mainnet*)
 
-Initial list of all on-chain assets owned by the protocol protected under Safe Harbor: [link](https://ipfs.io/ipfs/QmZ3xk9qa2uBmj9Si4Kv98PjTehxr4QyiMnmKSd9dPg6Ln) (all contract addresses can be verified using [Lido Docs](https://docs.lido.fi/deployed-contracts/)). The initial list may be updated by the Safe Harbor Management committee until ownership transfer to Aragon Voting, with all changes published on the research forum in the [dedicated thread](https://research.lido.fi/t/proposal-adopt-the-seal-safe-harbor-agreement/10940/11).
+Initial list of all on-chain assets owned by the protocol protected under Safe Harbor can be found in an [associated Snapshot proposal](https://snapshot.box/#/s:lido-snapshot.eth/proposal/0x0188ab77d59b11ce589d88c350093faffdb07b3a9c9ba4d8af12755d4b2178c0).
+
+As the protocol evolves, new contracts will be reviewed and added to the Safe Harbor Agreement scope, ensuring continued protection for all new contracts and functionalities.
+
+An up-to-date list of contracts under a scope of program can be found in a [Safe Harbor Agreement Contract](https://etherscan.io/address/0xe19f54e8322214839a87408f084aa14ebefe9e87).
```

Comment:
Can you actually call the getDetails method on Etherscan and see results? For me it's not working
This one works though: https://eth.blockscout.com/address/0xE19F54E8322214839A87408f084aA14eBEfe9E87?tab=read_contract#0xfbbf93a0